go/kbnm/main.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/qrtz/nativemessaging"
 )
 
+// Version is the build version of kbnm, overwritten during build.
+const Version = "dev"
```

Comment:
This is to differentiate when it was built from `go build` vs through our script (which overrides the variable with ldflags). I think `0.1.1` might be not as obvious?